web-ui/src/main/resources/catalog/components/viewer/wfsfilter/WfsFilterDirective.js

```diff
@@ -991,7 +980,7 @@
           }
 
           // check whether the WFS service is already in the database
-          scope.messageProducersApiUrl = gnHttp.getService('wfsMessageProducers');
+          scope.messageProducersApiUrl = '../api/msg_producers';
```

Comment:
Maybe we should add the missing entries here? https://github.com/geonetwork/core-geonetwork/blob/6991fffabc8566339c48cb8b23b919d259dbe9b8/web-ui/src/main/resources/catalog/components/catalog/CatalogService.js#L321

Instead of writing down a hardcoded url.

Reply:
The `gnHttpServices` 
https://github.com/geonetwork/core-geonetwork/blame/4.0.x/web-ui/src/main/resources/catalog/components/catalog/CatalogService.js#L299-L322 has not been modified in the last 2 years, so probably not the main way we are using for accessing the API (even if it was probably a good idea 7 years ago).

Reply:
Ok, no worries